app/scripts/views/sign_up.js

```diff
@@ -169,10 +169,14 @@ function (Cocktail, p, BaseView, FormView, Template, AuthErrors, mailcheck,
 
       var relier = this.relier;
       var isSync = relier.isSync();
+
+      var showChooseWhatToSyncCheckbox =
+        this.broker.hasCapability('showChooseWhatToSyncCheckbox') &&
```

Comment:
I think `showChooseWhatToSyncCheckbox` should be encoded into the brokers as only the `showChooseWhatToSyncCheckbox` capability. For brokers that have `chooseWhatToSyncWebV1` (currently only Fennec), `showChooseWhatToSyncCheckbox` would be false. 

Doing so gives us the flexibility to allow us to re-introduce the checkbox for Fennec users and only show the choose_what_to_sync screen if the user clicked the checkbox. All logic to make that decision would be in the broker's [afterSignUp](https://github.com/mozilla/fxa-content-server/blob/342c56cf67e09613688b410406c574154f38a1cd/app/scripts/models/auth_brokers/fx-fennec-v1.js#L49) function and the view would remain ignorant/unchanged.